students/kyle_lehning/Lesson07/html_render.py

```diff
@@ -23,24 +27,30 @@ def __init__(self, content=None, **kwargs):
     def append(self, new_content):
         self.contents.append(new_content)
 
-    def render(self, out_file):
```

Comment:
this comment is actually for line 23

```
>>> def test(**kwargs): print(kwargs)
...
>>> test()
{}
```
you dont need the `if kwargs.__len__() is not 0:` check at all .. and normally you would check via `if kwargs:`

you also shouldn't be using `__len__` magic method but instead use `len()` method.

as you can see in my above example kwargs is `{}` when nothing is passed in so you can safely use for loop over empty dictionary.